scripts/gendeclarations.lua

```diff
@@ -220,6 +231,7 @@ while init ~= nil do
       else
         non_static = non_static .. declaration
       end
+      init = e
```

Comment:
Added this in the last push: We had found the start of a function defintion (that ended at `e`), so we can continue searching for a new definition after this one.